R/transform_log.R

```diff
@@ -0,0 +1,19 @@
+#' transform_log 
+#' Transform numerical values into their log values
+#' @param x A vector
+#' @return logarithm of x
+#' @export 
+#' @examples
+#' transform_log(1)
+#' transform_log(c(1, 2, 3, 4, 5))
+#' 
+transform_log <- function(x) { 
+  if (!is.numeric(x)) {
+    warning("transform_log: Expecting numeric argument")
```

Comment:
You should stop here instead of indicating a warning. Implicit data type conversions can always lead to strange results.